devtools/etdump/tests/etdump_test.cpp

```diff
@@ -821,7 +821,7 @@ TEST_F(ProfilerETDumpTest, LogDelegateEvents) {
     etdump_gen[i]->log_profiling_delegate(
         nullptr, 278, 1, 2, metadata, strlen(metadata) + 1);
     EventTracerEntry entry = etdump_gen[i]->start_profiling_delegate(
```

Comment:
Here please check the error status first by `.ok()` then fetch its content by `.get()`